stakingcontract/sc.go

```diff
@@ -198,6 +228,15 @@ func (c ContractClient) GetVoters(opts *bind.CallOpts) ([]common.Address, error)
 	return response, nil
 }
 
+// GetVoterStake returns the staking of a voter from SC
+func (c ContractClient) GetVoterStake(opts *bind.CallOpts, voter common.Address) (*big.Int, error) {
```

Comment:
Should use pointer receiver func `func (c *ContractClient)`
Please update for another function.

Reply:
ok updated